inlineplz/main.py

```diff
@@ -65,6 +65,7 @@ def main():
     result = inline(args)
     print("inline-plz version: {}".format(__version__))
     print("Python version: {}".format(sys.version))
+    # TODO: This time is shorter than the longest running linter task in Travis CI somehow??
```

Comment:
thats odd

Reply:
I haven't seen this manifest ever since I added in the timeouts, but something to keep track of if it crops up again.